docs/rum_collection/crash_reporting.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Crash Reporting (beta)
+kind: documentation
+further_reading:
+  - link: "https://github.com/DataDog/dd-sdk-ios"
+    tag: "Github"
+    text: "dd-sdk-ios Source code"
+  - link: "/real_user_monitoring"
+    tag: "Documentation"
+    text: "Datadog Real User Monitoring"
+---
+
+Enable iOS crash reporting and error tracking to get comprehensive crash reports, and error trends in RUM UI. With this beta feature, you get access to 
+
+ - Aggregated iOS crash data and RUM crash attributes
+ - Desymbolicated iOS error reports
+ - Trend analysis with iOS error tracking
+
+This guide follows the setup for iOS crash reporting and error tracking in the following steps
+ - Add crash reporting
+ - Desymbolicate error reports
+ - Verify setup
+
+### Add crash reporting 
+
+If you have not set up the SDK yet, follow the [in-app setup instructions][1] or refer to the [iOS RUM setup documentation][2]. Upgrade to [dd-sdk-ios v1.7.0+][3] to get access to iOS crash reporting and error tracking. Update your initialization snippet to include crash reporting:
+
+
```

Comment:
As `DatadogCrashReporting` is a separate library, we also need instructions on adding it to the project. I'd rather prefer having a separate doc dedicated fully to `Datadog` and `DatadogCrashReporting` installation - there are many nuances we could additionally explain. Dependencies installation on iOS is very diversed, sometimes tricky and never seamless 🙁 , that's why most other products have dedicated content for this.

For now, we could just mention it shortly here and we could discuss providing separate installation doc later:

> Add `DatadogCrashReporting` dependency to your project. If you use Cocoapods use `pod 'DatadogSDKCrashReporting'`. For SPM and Carthage `DatadogCrashReporting` is available with `dd-sdk-ios`.
> ```
> | Package manager            | Installation method                                                                         |
> |----------------------------|-------------------------------------------------------|
> | CocoaPods                  | `pod 'DatadogSDKCrashReporting'`                      |
> | Swift Package Manager      | link `DatadogCrashReporting` module                   |
> | Carthage                   | use `DatadogCrashReporting.xcframework`               |
> ```

Reply:
+1, am happy to scope out this doc work and create a DOCS ticket in Jira when I'm back from PTO!